### PR TITLE
Improve RTL support

### PR DESCRIPTION
**Pull request by abarth**
Fix a bunch of obvious RTL bugs from code inspection.

### CODE REVIEW

File: packages/flutter/lib/src/painting/flutter_logo.dart [hunk 1]
```diff
@@ -12,6 +12,7 @@ import 'package:flutter/foundation.dart';
 import 'basic_types.dart';
 import 'box_fit.dart';
 import 'decoration.dart';
+import 'edge_insets.dart';
```
Comment (Hixie) [inline packages/flutter/lib/src/painting/flutter_logo.dart]: ?

**Comment by Hixie** [comment]
We really should test a lot of these...
LGTM

File: packages/flutter/lib/src/painting/flutter_logo.dart (hunk 1 above)
Comment (abarth) [reply packages/flutter/lib/src/painting/flutter_logo.dart]: Maybe something (decoration.dart?) exports EdgeInsets but not EdgeInsetsDirectional?  The analyzer was made without this import.

Comment (Hixie) [reply packages/flutter/lib/src/painting/flutter_logo.dart]: But you didn't make any other changes to this file. I don't understand.

Comment (abarth) [reply packages/flutter/lib/src/painting/flutter_logo.dart]: Ah, sorry.  I tried to RTL-ize FlutterLogoDecoration and backed out.  I'll remove this diff.

**Pull request merged**